src/index.js

```diff
@@ -1 +1,16 @@
-'use strict';

const scale = require('./mode');
const chord = require('./chord');
const clip = require('./clip');
const pattern = require('./pattern');
const midi = require('./midi');
const scales = require('./modes');

let modes = Object.keys(scales);

// Allow scale to be denoted by mode as well
module.exports = {mode: scale, scale, chord: chord.getChord, listChords: chord.listChords, modes, scales: modes, clip, pattern, midi};
\ No newline at end of file
+'use strict';
+const scale = require('./mode');
+const chord = require('./chord');
+const clip = require('./clip');
+const pattern = require('./pattern');
+const midi = require('./midi');
+const scales = require('./modes');
+function setMiddle(octaveIndex){
+    midi.setMiddle(octaveIndex);
+    //Trans is the positive octave transposition
+    //If trans is 1, all notes will be shifted up one octave
```

Comment:
Add a space after `//` for the above 2 lines, so that it look like
```
// Trans is the positive octave transposition
// If trans is 1, all notes will be shifted up one octave
```